health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/pages/employee/SearchChecklist.js

```
@@ -73,27 +73,26 @@ const SearchChecklist = () => {
-        isActive: true
-      }
+        filters: { type: "DEFAULT" },
+        isActive: true,
+      },
     },
-    changeQueryName: "HCM"
+    changeQueryName: "HCM",
   };
   const { isLoading, data: HCM, isFetching } = Digit.Hooks.useCustomAPIHook(reqCriteria1);
   useEffect(() => {
     let data = HCM?.mdms;
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'data' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)